ui/src/link_table_cell_type.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { S } from 'h2o-wave'
+
+export interface LinkTableCellType {
```

Comment:
Make sure the interface and implementation for cell are colocated (for context). So either move this into `table.tsx` or move the cell implementation here. Whatever you prefer.

Reply:
Moved